include/picongpu/plugins/openPMD/Json.cpp

```diff
@@ -339,6 +339,37 @@ The key 'select' must point to either a single string or an array of strings.)EN
          * careful memory configuration is necessary anyway, so the
          * defaults don't matter.
          */
+#    if OPENPMDAPI_VERSION_GE(0, 16, 0)
+        /*
+         * In openPMD >= 0.16, additionally ignore all attribute metadata
+         * that comes from a rank other than rank 0.
+         * The openPMD plugin of PIConGPU writes attributes synchronously
+         * (since this is required in common configurations of HDF5).
+         * For ADIOS2 however, it's better to write attributes from single
+         * ranks only and avoid metadata duplication across ranks.
+         * This option tells the ADIOS2 backend of openPMD that it can
+         * safely ignore any attribute write if the current MPI rank is
+         * not 0.
+         */
+        std::string const& defaultValues = R"(
+{
+  "hdf5": {
+    "dataset": {
+      "chunks": "none"
+    }
+  },
+  "adios2": {
+    "attribute_writing_ranks": 0,
```

Comment:
Why isn't this more targeted? Like this
```suggestion
#    if OPENPMDAPI_VERSION_GE(0, 16, 0)
    "attribute_writing_ranks": 0,
#    endif
```
(with appropriate string concatenation). The next person changing something in this configuration will have to do so in two places. I'm aware that these are close together and it's not a huge burden but still it's something that could get overlooked in a quick diff that doesn't show the other section. (In fact, I've only seen the duplication because I've looked at the full file to understand what's going on. The current diff doesn't even show the other version.)